docs/sources/tutorials/get-started.md

```diff
@@ -71,17 +87,168 @@ services:
+
+```alloy
+local.file_match "local_files" {
+    path_targets = [{"__path__" = "/var/log/*.log"}]
+    sync_period = "5s"
+}
+```
+
+This component creates a [local.file_match][] component named `local_files` with an attribute that tells {{< param "PRODUCT_NAME" >}} which files to source, and to check every 5 seconds.
+
+### Second Component: Scraping
+
+Paste this component next in the `config.alloy` file:
+
+```alloy
+loki.source.file "log_scrape" {
+    targets    = local.file_match.local_files.targets
+    forward_to = [loki.write.grafana_loki.receiver]
+    tail_from_end = true
+}
+```
+
+This configuration creates a [loki.source.file][] component named `log_scrape`, and shows the pipeline concept of {{< param "PRODUCT_NAME" >}} in action. The `log.scrape` component does the following:
+
+1. It applies to the `local_files` component (its "source" or target).
+1. It forwards the logs it scrapes to the "receiver" of another component called `grafana_loki` that you will define next.
+1. It provides extra attributes and options, in this case, you will tail log files from the end and not ingest the entire past history.
+
+### Third Component: Write Logs to Loki
+
+Paste this component last in your configuration file:
+
+```alloy
+loki.write "grafana_loki" {
+  endpoint {
+    url = "http://localhost:3100/loki/api/v1/push"
+
+    // basic_auth {
+    //  username = "admin"
+    //  password = "admin"
+    // }
+  }
+}
+```
+
+This last component creates a [loki.write][] component named `grafana_loki` that points to `http://localhost:3100/loki/api/v1/push`.
+This completes the simple configuration pipeline.
+
+{{< admonition type="tip" >}}
+The `basic_auth` is commented out because the local `docker compose` stack doesn't require it. 
+It is included in this example to show how you can configure authorization for other environments.
+For further authorization options, refer to the [loki.write][] component reference.
+
+{{< /admonition >}}
+
+This connects directly to the Loki instance running in the Docker container.
+
+## Reload the Configuration
+
+Copy your local `config.alloy` file into the default configuration file location, which depends on your OS.
+
+{{< code >}}
+
+```macos
+sudo cp config.alloy $(brew --prefix)/etc/alloy/config.alloy
+```
+
+```linux
+sudo cp config.alloy /etc/alloy/config.alloy
+```
+
+{{< /code >}}
+
+Finally, call the reload endpoint to alert {{< param "PRODUCT_NAME" >}} to the configuration change without the need
+for restarting the system service.
+
+```bash
+    curl -X POST http://localhost:12345/-/reload
+```
+
+{{< admonition type="tip" >}}
+This step uses the Alloy UI, which is exposed only on `localhost` port `12345`. If you chose to run Alloy in
+a dockerized environment, you will need to expose this to other addresses, for example via the argument
+`--server.http.listen-addr=0.0.0.0:12345` when running {{< param "PRODUCT_NAME" >}}
+For more information refer to [debugging your configuration]
```

Comment:
```suggestion
This step uses the Alloy UI, which is exposed on `localhost` port `12345`.
If you chose to run Alloy in a Docker container, make sure you use the `--server.http.listen-addr=0.0.0.0:12345` argument.
If you don’t use this argument, the [debugging UI][debug] won’t be available outside of the Docker container.

[debug]: ../../tasks/debug/#alloy-ui
```

Aligning the text with the note here: https://grafana.com/docs/alloy/latest/get-started/install/docker/#run-a-linux-docker-container